File: add.c
#include<stdio.h>

void getinput(int *a, int *b)
{
printf("Enter two numbers:");
scanf("%d %d ",a,b);

}
int add()
{
 int a , b ,sum;
sum=a+b;
}
int getoutput()
{
printf("%d+%d=%d",a,b,sum);

}
int main()
{
int a,b,c,sum;
 getinput(&a,&b);
 sum(&a,&b);
getoutput(&a,&b,&c,&sum);
}

File: add.py
def getinput(a,b):
    a=int(input('enter a:'))
    b=int(input('enter b:'))
    return a,b
    
def add(a,b):
    c=a+b
    return c
def getoutput(c):
    print("sum=",c)
def main():
    a=0;b=0;c=0;
    a,b=getinput(a,b)
    c=add(a,b)
    getoutput(c)
main()
    
